package.json

```diff
@@ -77,31 +77,31 @@
     ]
   },
   "dependencies": {
-    "@angular/animations": "^7.0.1",
+    "@angular/animations": "^8.0.0-beta.10",
```

Comment:
Do we also have to update the package version number?

Reply:
Not sure what you mean?

Reply:
Sorry, I meant this line https://github.com/ngrx/platform/blob/master/package.json#L3